chain/chain/src/state_snapshot_actor.rs

```diff
@@ -14,8 +14,7 @@ use std::sync::Arc;
 /// Runs tasks related to state snapshots.
 /// There are three main handlers in StateSnapshotActor and they are called in sequence
 /// 1. DeleteSnapshotRequest: deletes a snapshot and optionally calls CreateSnapshotRequest.
```

Comment:
nit: It's now called `DeleteAndMaybeCreateSnapshotRequest`. Not your code but close enough for a tiny fix :)

Reply:
Also you can use the ```[`DeleteAndMaybeCreateSnapshotRequest`]``` syntax to make it actually reference the real class with full IDE support.